Dockerfile

```diff
@@ -21,13 +21,18 @@ RUN pecl channel-update pecl.php.net \
     && docker-php-ext-enable \
         apcu
 
+COPY docker/conf.d /usr/local/etc/php/conf.d
+
 RUN apt-get autoremove --purge -y && apt-get clean
 
-COPY docker/conf.d /usr/local/etc/php/conf.d
+RUN php -r "copy('http://getcomposer.org/installer', 'composer-setup.php');"
+RUN php composer-setup.php --install-dir=/usr/local/bin --filename=composer
+RUN php -r "unlink('composer-setup.php');"
 
 COPY . /app
 
 WORKDIR /app
 
-CMD ["php", "/app/build.php"]
+RUN composer install
 
+ENTRYPOINT ["php", "/app/build.php"]
```

Comment:
Using `ENTRYPOINT` would allow you to later add options to `build.php` by executing `build.sh --param1 --param2`. This will be amended to `["php", "/app/build.php"]`. see https://phoenixnap.com/kb/docker-cmd-vs-entrypoint